src/content/docs/extensions/algos/index.mdx

```diff
@@ -0,0 +1,219 @@
+---
+title: Graph algorithms
+---
+
+The graph algorithms extension package allows you to directly run popular graph algorithms
+like PageRank, connected components and Louvain on your graph data stored in Kuzu. Using this extension,
+you do not need to export your data to specialized graph analytics tools like NetworkX (at
+least for the algorithms that are supported by the extension). The algorithms run natively
+in Kuzu, which also allows you to scale to very large graphs!
+
```

Comment:
Should we also link to "(weighted) shortest path" here? and explain that they're built-in provided.

Reply:
Added though I don't know what we should explain about built-in. I just point to the pattern matching doc for shortest path.